Dockerfile

```diff
@@ -1,6 +1,27 @@
 FROM docker:18.09.1 AS docker
-FROM docker/compose:1.23.2
-RUN apk --no-cache add make git bash curl 
+
+FROM ubuntu:18.04
+
+# install docker client
 COPY --from=docker /usr/local/bin/docker /usr/local/bin/docker
+
+# install golang & ruby
+RUN apt-get -y update
+RUN apt-get install -y build-essential git-core curl
+RUN apt-get install -y golang-go
+RUN apt-get install -y ruby-full
+
+# install nodejs
```

Comment:
What do we need node and Ruby for?

Reply:
This way, people using node or ruby can execute commands right away